dns/dnsperfgo/main.go

```diff
@@ -223,6 +235,12 @@ func (c *dnsClient) runQuery(name string, timeout time.Duration, lookupFunc Look
 		startTime := time.Now()
 		_, err := lookupFunc(name)
 		latency := time.Since(startTime)
+		if infoLogger != nil && err == nil {
```

Comment:
why not route the infoLogger to a dummy logger that doesn't output instead of having `if` statements?

Reply:
Thanks. 
After my change, now the infoLogger will now record the successful lookup and error (e.g. no such host). To measure latency accurately, the time out will not reach the infoLogger. But the errorLogger will record error (e.g. no such host) and time out. I think it is enough to cover all the information we need.